### DISCUSSION TITLE
Using `openpnm` on huge networks (~10^6 pores)

### DISCUSSION THREAD

**Discussion by Mat10G** [PMEAL/OpenPNM#2508] [Q&A]
Hello everyone, is it possible to use OpenPNM for transport works in large aquifers (several km)? Can calculations be performed on very large networks (millions of pores) ?

**Comment by jgostick** [comment]
Millions of pores, with a spacing of 100 um per pore is still only a few cm.  Km simulations with pore scale resolution are not feasible.  On the other hand, it is possible to use OpenPNM a basic continuum solver, where you put a porosity value in each pore, which is more like a 'node' or block in the reservoir.  You can then find the permeability and capillary pressure behavior from correlations, then model transport even two phase flow.  We don't have any examples on this, but it is possible.
